src/gui_widgy_editor.c

```diff
@@ -862,24 +862,33 @@ on_combo_changed(GtkComboBox *widget,
 }
 
 static void
-place_combo(GtkObject *adj, GtkWidget *table, int x, int y, int type)
+combo_box_append_text(GtkWidget *w, const char** labels)
 {
-    const char **labels;
     int i;
-    /* -FIX- this is kinda ugly */
+    for (i = 0; labels[i]; i++)
+    {
+        gtk_combo_box_append_text(GTK_COMBO_BOX(w), labels[i]);
+    }
+}
+
+static void
+place_combo(GtkObject *adj, GtkWidget *table, int x, int y, int type)
+{
+    /* -FIX- this is still kinda ugly */
+    const char* labelsLfoWave[] = { "Tri", "Saw+", "Saw-", "Square", "Sine", "S/H", 0 };
+    const char* labelsCurve[] = { "-Lin", "-Exp", "+Exp", "+Lin", 0 };
```

Comment:
For consistency with the rest of the code, I suggest the snake_case names labels_lfo_wave and labels_curve, instead of the camelCase names labelsLfoWave and labelsCurve.

Reply:
Fix:
Patching segfault with patch from https://salsa.debian.org/multimedia-team/hexter/-/commit/799ea5ddcbfeb0fb3d83b54a0a08702175699496

modified as suggested in snake_case names for labels_lfo_wave and labels_curve
in the file fix-segfault.patch

correct the m4 and subdir error in the file fix-configureac.patch 

Patch available in report for ArchLinux https://bugs.archlinux.org/task/67872

Reply:
Thank you jmechmech, that works for me too♥

Reply:
Thanks everyone for providing the fix! I just rolled it into release 1.1.1.